### ISSUE TITLE
Instead of enablig/disablin yum repos we should use whatever is enabled

### ISSUE THREAD

**Issue by goldmann** [concrt/concreate#104]
Currently we disable every repository in the system and explicitly enable the one we want. We should use all enabled repositories by default. This will greatly enhance usability of this tool for upstream projects.

**Comment by dbecvarik** [comment]
+1 this is really good idea.

**Comment by dbecvarik** [comment]
our current state for installing packages and enabling repos is:
``` yaml:
packages:
      repositories:
          - repo1
          - repo2
install:
          - pkg1
```

I was thinking about introducing following syntax:
``` yaml:
packages:
      repositories:
          - repo1
            state: enabled
            url: 'http://foo.bar.bz' # it's possible to use cacher here to hide url
          - repo2
            state: disabled
install:
          - pkg1
```
We only need to think how we can hide URL and/or fetch repositories. I'll work on this more  - I dont much like current state where repo is silently defined  and fetched inside concreate config file

**Comment by dbecvarik** [comment]
this was fixed by #187
